README.md

```diff
@@ -104,6 +105,13 @@ The AWS IoT Device Defender library has no dependencies on additional libraries
 
 See memory requirements for the latest release [here](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/device-defender-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#defender_memory_requirements).
 
+#### AWS IoT Over-The-Air Updates
+The [AWS IoT Over-The-Air Updates](https://github.com/aws/OTA-for-AWS-IoT-embedded-SDK) library enables you to manage the notification of a newly available update, download the update, and perform cryptographic verification of the firmware update. Using the over-the-air (OTA) client library, you can logically separate firmware updates from the application running on your devices. Although OTA updates were designed to update device firmware, you can use them to send any files to one or more devices registered with AWS IoT.  More details about OTA library can be found in [AWS IoT Over-The-Air Updates documentation](https://docs.aws.amazon.com/iot/latest/developerguide/iot-ota.html).
```

Comment:
The AWS IoT Over-the-air Updates (OTA) library enables you to manage the notification of a newly available update, download the update, and perform cryptographic verification of the firmware update. Using the OTA library, you can logically separate firmware updates from the application running on your devices. You can also use the library to send other files (e.g. images, certificates) to one or more devices registered with AWS IoT.

Reply:
Thanks, updated in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1454/commits/1aa621d336055f201cddce7f917f67f38d609df2